cypress/integration/frontpageLayoutSpec.js

```diff
@@ -0,0 +1,146 @@
+import services from '../support/worldServices';
+import { el } from '../support';
+import describeForLocalOnly from '../support/describeForLocalOnly';
+
+Object.keys(services).forEach(index => {
+  const xservice = services[index];
+  const service = index;
+
+  // TODO this needs to become a normal describe block once Mozart routing is set up
+  describeForLocalOnly(`frontpage tests for ${service}`, () => {
+    // eslint-disable-next-line no-undef
+    before(() => {
+      cy.visit(xservice.url);
+    });
+
+    describe('checks the components are present', () => {
+      // eslint-disable-next-line no-undef
+      before(() => {
+        cy.viewport(1008, 768);
+      });
+
+      describe('cookie banner', () => {
+        it('should have a visible cookie banner', () => {
+          cy.get(el.cookieBanner).should('be.visible');
+        });
+
+        it('should have a functional cookie banner', () => {
+          cy.get(el.cookieBanner).within(() => {
+            cy.get('a')
+              .should('have.length.of', 1)
+              .should(
+                'have.attr',
+                'href',
+                'https://www.bbc.co.uk/usingthebbc/your-data-matters',
+              );
+            cy.get('button')
+              .should('have.attr', 'type', 'button')
+              .click();
+          });
+
+          cy.get(el.cookieBanner).within(() => {
+            cy.get('a')
+              .should('have.length.of', 2)
+              .should(
+                'have.attr',
+                'href',
+                'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
+              )
+              .last()
+              .should(
+                'have.attr',
+                'href',
+                'https://www.bbc.co.uk/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+              );
+
+            cy.get('button')
+              .should('have.attr', 'type', 'button')
+              .click();
+          });
+        });
+      });
+
+      describe('header tests', () => {
+        it('should have a visible banner', () => {
+          cy.get(el.header)
+            .should('have.lengthOf', 1)
+            .should('have.attr', 'role', 'banner')
+            .find('a')
+            .should('have.attr', 'href', 'https://www.bbc.co.uk/news') // this should one day soon become 'https://www.bbc.com/igbo' etc.
```

Comment:
Can you pull this value from config?

Reply:
Have had a chat with Drew, we're going to leave this for now and fix it once the team has decided how to merge test data/configs.